docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -125,26 +268,25 @@ export default defineComponent({
     dark: {
       type: Boolean,
       default: false
+    },
+    scrollData: {
+      type: Object,
+      default: () => ({})
     }
   },
   components: {
-    NavDropdownMenu,
-    AppSearchResults
+    HeaderNavLink,
+    SearchQuasarForm,
+    NavDropdownMenu
   },
   setup (props) {
     const $q = useQuasar()
-    const $route = useRoute()
     const isSearchFieldActive = ref(false)
     const showNavItems = ref(true)
-
-    function toggleSearchInputField () {
-      if (isSearchFieldActive.value) {
-        showNavItems.value = true
-      }
-      else showNavItems.value = !Screen.lt.lg
-
-      isSearchFieldActive.value = !isSearchFieldActive.value
-    }
+    const searchForm = ref(null)
```

Comment:
Same questions as the other `ref` initialized with `null`

Reply:
For a similar reason as noted above.